File: sms-status/php/status.php
<?php
require_once('Vpbx.php');

define ('VPBX_API_URL', 'https://api.vpbx.pl');

// Uźytkownik i hasło wygenerowane w portalu https://ssl.vpbx.pl
define ('VPBX_API_USERNAME', '');
define ('VPBX_API_PASSWORD', '');

//SMS ID
$sms_id = "";


$vpbx = new Vpbx();

$token_response = $vpbx->Post(VPBX_API_URL . "/api/v1/login", array("username"=> VPBX_API_USERNAME, "password"=> VPBX_API_PASSWORD));
if(isset($token_response['token'])){
	$vpbx->SetAuth($token_response['token']);
	$response = $vpbx->Get(VPBX_API_URL . "/api/v1/sms/" . $sms_id);
        print_r($response);
} else {
	echo "Generowanie tokenu nie powiodło się. Przenalizuj błąd:\n";
	print_r($token_response);
}

?>

File: sms-status/python/status.py
import requests


api_url = "https://api.vpbx.pl/api/v1"
#API Username
username = ""
#API Password
password = ""

#SMS ID 
sms_id = ""

reqToken = requests.post(api_url + "/login", json={"username": username, "password": password})

if reqToken.status_code != 200:
    print("Failed to authenticate. Status : " + reqToken.json()["result"] + ",  Error: " + reqToken.json()["error"])
    exit(1)

token = reqToken.json()["token"]


respObject = requests.get(api_url + "/sms/" + sms_id, headers={"authorization": "Bearer " + token})

if respObject.status_code != 200:
    print("Failed to send the sms. Status : " + respObject.json()["result"] + ",  Error: " + respObject.json()["error"])
    exit(1)

print("Status : " + respObject.json()["result"] + ",  Status : " + respObject.json()["status"])



